libs/application/templates/login-service/src/fields/IsatInvalid/IsatInvalid.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { FieldBaseProps, formatText } from '@island.is/application/core'
+import {
+  AlertMessage,
+  Box,
+  GridColumn,
+  Link,
+  Text,
+} from '@island.is/island-ui/core'
+import { useLocale } from '@island.is/localization'
+import { errorMessages } from '../../lib/messages'
+
+export const IsatInvalid = ({ application }: FieldBaseProps) => {
+  const { formatMessage } = useLocale()
+  return (
+    <>
+      <GridColumn span={['1/1', '1/1', '1/1']} paddingTop={2}>
```

Comment:
`span` accepts a single value and as the value is always the same you can do instead
```suggestion
      <GridColumn span='1/1' paddingTop={2}>
```